services/core-web/src/components/noticeOfWork/applications/NoticeOfWorkApplication.js

```diff
@@ -337,8 +358,13 @@ export class NoticeOfWorkApplication extends Component {
           <div className="inline-flex between">
             <div>
               <h1>NoW Number: {this.props.noticeOfWork.now_number || Strings.EMPTY_FIELD}</h1>
+              {this.state && this.state.fromRoute && (
```

Comment:
this.state will always exist on a component, so probably not necessary to add safety around it (but doesn't hurt either).

Reply:
I didn't know that it would, thanks!